app/controllers/cart_staches_controller.rb

```diff
@@ -10,6 +11,7 @@ def destroy
     stache = Stache.find(params[:id])
     @cart.delete_stache(stache.id)
     session[:cart] = @cart.contents
-    redirect_to cart_path
+    flash.now[:removed] = "Successfully removed #{view_context.link_to stache.name, stache_path(stache)} from your cart.".html_safe
+    render 'carts/show'# redirect_to cart_path
```

Comment:
Can we change this back to `redirect_to cart_path` since we know our flash messages are in fact working now?